src/DoctrineBridge/Router/EntityIdResolver.php

```diff
@@ -4,7 +4,7 @@
 
 use Doctrine\Common\Persistence\ManagerRegistry;
 use Doctrine\ORM\Mapping\MappingException;
-use Iltar\HttpBundle\Exception\UncallableMethodException;
+use Iltar\HttpBundle\DoctrineBridge\IdFromEntityInterface;
```

Comment:
Indeed, I will move this. This can be re-used easily by Eloquent, propel or any custom ORM which uses an id. 

Now that I think of it, I could move the whole IdResolver out of the bridge and let the ORM specific implementation handle the id resolving. I should probably add an `isManaged($entity)` to the `IdFromEntityInterface`. In this case I could also call it `ManagedModelInterface` or something, any suggestions on that?